src/unistd.rs

```diff
@@ -1132,6 +1132,57 @@ pub fn isatty(fd: RawFd) -> Result<bool> {
     }
 }
 
+
```

Comment:
Too many blank lines here.

Reply:
Hi ... didn't notice these comments. Will remove.

Reply:
You still haven't removed them.  Please don't mark conversations as resolved unless you've actually resolved them.